src/writers.ts

```diff
@@ -1,12 +1,78 @@
 import { evm } from '@snapshot-labs/checkpoint';
-import { Payment } from '../.checkpoint/models';
+import { Payment, Space } from '../.checkpoint/models';
 import { getJSON } from './utils';
 import tokens from './payment_tokens.json';
 
+const MILLISECONDS = 1000;
+const DECIMALS = 1e6; // USDC and USDT both have 6 decimals
+
+const TURBO_MONTHLY_PRICE = 1 * DECIMALS; // 600 USDC | USDT
```

Comment:
I would prefer to have this also customizable via environment variables, so we could override it on testnet with smaller amount, without updating the code

Reply:
3958ae823fe67b66203f2fd2934deabbb59a63ae good idea! done

Reply:
didn't put the `decimals` as env vars because at the end of the day it's just a matter of adding `0` to your tests or not, should be easy. But changed the yearly and monthly to env vars